analyzers/tests/SonarAnalyzer.UnitTest/Rules/NonDerivedPrivateClassesShouldBeSealedTest.cs

```diff
@@ -0,0 +1,41 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2021 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+using System.Collections.Generic;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SonarAnalyzer.Rules.CSharp;
+using SonarAnalyzer.UnitTest.TestFramework;
+
+namespace SonarAnalyzer.UnitTest.Rules
+{
+    [TestClass]
+    public class NonDerivedPrivateClassesShouldBeSealedTest
+    {
+        private readonly List<string> testFiles = new List<string>
+        {
+            @"TestCases\NonDerivedPrivateClassesShouldBeSealed.cs",
+            @"TestCases\NonDerivedPrivateClassesShouldBeSealed_PartialClass.cs"
+        };
+
+        [TestMethod]
+        [TestCategory("Rule")]
+        public void NonDerivedPrivateClassesShouldBeSealed_CS() =>
+            Verifier.VerifyAnalyzer(testFiles, new NonDerivedPrivateClassesShouldBeSealed());
```

Comment:
We tend to add the file paths inside the test since we could have multiple tests which usually use different paths.

```suggestion
public void NonDerivedPrivateClassesShouldBeSealed_CS() =>
            Verifier.VerifyAnalyzer(new[] { @"TestCases\NonDerivedPrivateClassesShouldBeSealed.cs", @"TestCases\NonDerivedPrivateClassesShouldBeSealed_PartialClass.cs" },
                                    new NonDerivedPrivateClassesShouldBeSealed());
```